MaterialLib/MPL/Properties/BishopsPowerLaw.cpp

```diff
@@ -0,0 +1,54 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "BishopsPowerLaw.h"
+
+namespace MaterialPropertyLib
+{
+BishopsPowerLaw::BishopsPowerLaw(double const exponent) : _m(exponent) {}
+
+void BishopsPowerLaw::setScale(
+    std::variant<Medium*, Phase*, Component*> scale_pointer)
+{
+    if (!std::holds_alternative<Medium*>(scale_pointer))
+    {
+        OGS_FATAL(
+            "The property 'BishopsPowerLaw' is implemented on the 'media' "
```

Comment:
media or medium scale?

Reply:
So far, only 'media' is used (due to copy-paste)...